client/src/components/graph/setupLasso.js

```diff
@@ -114,6 +115,22 @@ const Lasso = () => {
         closePath = null;
       }
     };
+
+    lasso.move = polygon => {
+      if (polygon !== lassoPolygon || polygon.length !== lassoPolygon.length) {
+        lasso.reset();
+
+        lassoPolygon = polygon;
+        lassoPath = g
+          .append("path")
```

Comment:
Please add to facilitate smoke testing:
```suggestion
          .append("path")
          .attr("data-testid", "lasso-element")
```

Reply:
added.